apps/full-site-editing/full-site-editing-plugin/newspack-blocks/index.php

```diff
@@ -64,14 +64,16 @@ function blog_posts_block_args( $args, $name ) {
 	wp_register_style( $block_prefix . 'editor', $editor_style, array(), NEWSPACK_BLOCKS__VERSION );
 
 	// View script.
-	$script_data = require NEWSPACK_BLOCKS__BLOCKS_DIRECTORY . $block_prefix . 'view.asset.php';
-	wp_register_script(
-		$block_prefix . 'view',
-		plugins_url( NEWSPACK_BLOCKS__BLOCKS_DIRECTORY . $block_prefix . 'view.js', __FILE__ ),
-		$script_data['dependencies'],
-		$script_data['version'],
-		true
-	);
+	if ( 'carousel' !== $name ) {
```

Comment:
Not sure this is the most elegant method but since we still only have two blocks here I didn't feel like a new abstraction would make things better.

I've tried to talk `calypso-build` into generating only CSS file and not JS but even if you remove all JS and you set build target directly to SCSS file, it still generates some JS so we can't rely on file presence as a check to enqueue and have to filter manually.